shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1334,9 +1335,10 @@ public void writeFields(DataOutput out) throws IOException {
             out.writeUTF(target);
             out.writeUTF(source);
             out.writeUTF(delegationToken == null ? "" : delegationToken);
-            out.writeInt(attributeUpdates.size());
-            for (ConditionalAttributeUpdate entry : attributeUpdates) {
-                entry.writeFields(out);
+            if (batch.isPresent()) {
```

Comment:
I am concerned that this will prevent us from expanding this command with new fields in the future. If we don't encode anything, then line 1350 below will be very confused should we decide to add anything else in the future. Let's encode a boolean indicating whether the batch info is present or not.

Reply:
Ok. will change this